tracim/tracim/templates/calendar/config.mak

```diff
@@ -43,7 +43,17 @@ var globalBackgroundSync=true;
 var globalSyncResourcesInterval=120000;
 var globalEnableRefresh=false;
 var globalEnableKbNavigation=true;
-var globalInterfaceLanguage=window.parent.globalTracimLang === 'fr' ? 'fr_FR' : window.parent.globalTracimLang; // globalTracimLang is declared in tracim header (caldavzap in in an iframe)
+// var globalInterfaceLanguage=window.parent.globalTracimLang === 'fr' ? 'fr_FR' : window.parent.globalTracimLang; // Côme - 2017/09/14 - line outdated
+// globalTracimLang is declared in tracim header (caldavzap in in an iframe)
+var globalInterfaceLanguage = (function (lang) {
+    switch (lang) {
+        case 'fr':
+            return 'fr_FR';
+        case 'en':
```

Comment:
Shouldn't we have a "default" case ? (in case `lang` has another content)
@Skylsmoi @algooapy talk to each other, decide and merge this PR.
Thanks